subsys/bluetooth/services/fast_pair/fp_storage/include/fp_storage_manager.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef FP_STORAGE_MANAGER_H_
+#define FP_STORAGE_MANAGER_H_
+
+/**
+ * @defgroup fp_storage_manager Fast Pair storage manager
+ * @brief The subsystem manages Fast Pair storage modules.
+ *
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @typedef fp_storage_manager_module_reset
+ * Callback used to reset Fast Pair storage module.
+ *
+ * @return 0 if the operation was successful. Otherwise, a (negative) error code is returned.
+ */
+typedef int (*fp_storage_manager_module_reset)(void);
+
+/**
+ * @typedef fp_storage_manager_reset_in_progress
+ * Callback used to inform Fast Pair storage module that reset operation is in progress.
+ *
+ * @return 0 if the operation was successful. Otherwise, a (negative) error code is returned.
+ */
+typedef int (*fp_storage_manager_reset_in_progress)(void);
```

Comment:
I would return `void` here. I think module should not be able to return error when notified about reset in progress.